README.md

```diff
@@ -83,3 +83,90 @@ Example:
+#### Install dependencies
+EUDAQ needs a recent [cmake3](https://cmake.org/download/) version that might not be shipped with your distribution. This is a [known issue](https://github.com/eudaq/eudaq/issues/466). To install a recent version under Ubuntu do
+```bash
+wget https://cmake.org/files/v3.11/cmake-3.11.1.tar.gz
+tar xf cmake-3.11.1.tar.gz
+cd cmake-3.11.1
+./configure
+make -j 4
+sudo apt-get install checkinstall
+sudo checkinstall
+```
+#### Install eudaq
+```bash
+git clone -b v1.x-dev https://github.com/eudaq/eudaq
+cd eudaq/build
+cmake -DBUILD_python=ON -DBUILD_gui=OFF -DBUILD_onlinemon=OFF -DBUILD_runsplitter=OFF -DUSE_ROOT=OFF ..
+make install -j 4
+```
+The producer has to know the installation path of EUDAQ. One way is to specify `PYTHONPATH` to include the `python` folder in the EUDAQ directory, e.g.:
+```bash
+export PYTHONPATH="${PYTHONPATH}:/home/user/git/eudaq/python"
+```
+This is not needed when you mention the path of the eudaq installation for every call to `pytlu_eudaq` (see below).
+
+**Run control GUI**
+
+When you want to have the run control GUI for more convenient testing change the CMAKE option to:
+```
+-DBUILD_gui=ON
+```
+Warning: you have to avoid calling `cmake` within the anaconda environment, since the anaconda QT5 version cannot easily be used to build code.
+
+### Usage with Pytlu
+A simple command line interface is provided to start the pytlu producer:
+```bash
+pytlu_eudaq --help
+```
+Please read the help output for program parameters.
+Since there are configuration parameters which exist within the [EUDAQ TLU controller](https://github.com/eudaq/eudaq/blob/v1.x-dev/producers/tlu/src/TLUController.cc) but not within the 
+pytlu producer and vice versa, only the following configuration parameters can be set using the EUDAQ config file:
+ 
+ - `TriggerInterval`
+ - `AndMask`
+ - `DutMask`
+
+These are mapped properly to the format needed by pytlu. Of course, the usual parameters supplied by pytlu can be specified, e.g.:
+
+```
+pytlu_eudaq --timeout 5  -f /path/to/data/
+```
+
+Please note that the settings specified in the EUDAQ configuration file will overwrite the settings specified by the command line interface.
+
+
+If you did not add the EUDAQ directory to the `PYTHONPATH` explicitly after installation (see above) you can give the path when running `pytlu_eudaq`, e.g.:
+
+```
+pytlu_eudaq --path /home/user/git/eudaq
+```
+
+### Debugging and testing
+#### Replay feature
+It is possible to replay a recorded pytlu raw data file with correct timing to test the system. This allows development and debugging without hardware. To replay a [pytlu raw data file](https://github.com/SiLab-Bonn/pytlu/blob/development/data/tlu_example_data.h5) one has to type:
```

Comment:
Is this file big, that one should use git lfs?

Reply:
< 200 kb

Reply:
Exactly, so I guess it is fine?

Reply:
sure